examples/cra-kitchen-sink/src/stories/issues.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+
+const hoc = WrapComponent => ({ ...props }) => <WrapComponent {...props} />;
+
+const Input = hoc(() => <input type="text" />);
+
+const TextArea = hoc(({ children }) => <textarea>{children}</textarea>);
+
+storiesOf('GitHub issues', module).add(
```

Comment:
Naming it github issues isn't very helpful. Maybe `Addon Info.HOC` and `withInfo('Allow Duplicate DisplayNames for HOC #1814')`?

Reply:
I wanted to introduce a single namespace for reproductions

Reply:
I understand, but ultimately, I still think this is a `Addon Info` thing. If we have 20 different `Github` issues and I'm making a change to `Addon Info`, I wouldn't think to check all the `Github` ones for regressions you know?

Maybe `Addon Info.Github Issues` and that description I had earlier?